os/board/sidk_s5jt200/src/s5jt200_tash.c

```diff
@@ -404,20 +403,9 @@ int board_app_initialize(void)
 		lldbg("Version Info :\n");
 		lldbg("tinyARA %s\n", __TIMESTAMP__);
 	}
-#if defined(CONFIG_RTC_DRIVER)
-	{
-		struct rtc_lowerhalf_s *rtclower;
-
-		rtclower = s5j_rtc_lowerhalf();
-		if (rtclower) {
-			ret = rtc_initialize(0, rtclower);
-			if (ret < 0) {
-				lldbg("Failed to register the RTC driver: %d\n",
-						ret);
-			}
-		}
-	}
-#endif /* CONFIG_RTC_DRIVER */
+#ifdef CONFIG_RTC_DRIVER
```

Comment:
yes actually previous is more correct way.

Reply:
Agree. Let me revert.